cmd/finch/main.go

```diff
@@ -32,6 +33,7 @@ func main() {
 	mem := fmemory.NewMemory()
 	stdOut := os.Stdout
 	if err := xmain(logger, stdLib, fs, stdLib, mem, stdOut); err != nil {
+		errutil.HandleExitCoder(err)
```

Comment:
Can we add a unit test for this?

Reply:
the package is from nerdctl, are we saying adding unit test for the Handle exit coder ?

Reply:
Not for the exit coder, but for the behavior change.

Reply:
may be integ test, dont see the point of adding unit test to main() function
Can add a test that runs a container and exits and we can see the status code. (similar to the test done locally)

Reply:
Added this test: https://github.com/runfinch/common-tests/pull/180
Once that is merged would rebase it.